lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -218,8 +221,15 @@ public function GetQueueContents(array $params)
             $typeidsfilter = ' AND t.type = '.$typeids;
         }
 
+	if($verifierids == 'all')
+		$verifierids=null;
+
```

Comment:
Jeśli cokolwiek nowego puszczasz to waliduj pod kątem zgodności z PSR-2. To na pewno nie będzie przechodzić walidacji...

Reply:
Travis to validuje?

Reply:
Z Travisem jest od paru dni jakiś problem - próbowałem to naprawić, ale coś się sypie i niestety przez to zostało przeoczonych parę niezgodności.